rootfs/usr/local/bin/launch.sh

```diff
@@ -76,7 +76,7 @@ if [ "$PHP82" = "true" ]; then PHP_INI_SCAN_DIR=/data/php/82/conf.d php-fpm82 -c
 if [ "$PHP83" = "true" ]; then PHP_INI_SCAN_DIR=/data/php/83/conf.d php-fpm83 -c /data/php/83 -y /data/php/83/php-fpm.conf -FOR; fi &
 if [ "$LOGROTATE" = "true" ]; then while true; do touch /data/etc/logrotate.lock; logrotate --verbose --state /data/etc/logrotate.state /etc/logrotate; rm /data/etc/logrotate.lock; sleep 25h; done; fi &
 # shellcheck disable=SC2086
-if [ "$GOA" = "true" ]; then while true; do if [ ! -f /data/etc/logrotate.lock ]; then goaccess --no-global-config --num-tests=0 --tz="$TZ" --date-format="%d/%b/%Y" --time-format="%H:%M:%S" --log-format='[%d:%t %^] %v %h %T "%r" %s %b %b %R %u' --no-ip-validation \
+if [ "$GOA" = "true" ]; then while true; do if [ ! -f /data/etc/logrotate.lock ]; then sleep 30s;  goaccess --no-global-config --num-tests=0 --tz="$TZ" --date-format="%d/%b/%Y" --time-format="%H:%M:%S" --log-format='[%d:%t %^] %v %h %T "%r" %s %b %b %R %u' --no-ip-validation \
```

Comment:
```suggestion
if [ "$GOA" = "true" ]; then while true; do if [ -f /data/nginx/access.log ] && [ ! -f /data/etc/logrotate.lock ]; then goaccess --no-global-config --num-tests=0 --tz="$TZ" --date-format="%d/%b/%Y" --time-format="%H:%M:%S" --log-format='[%d:%t %^] %v %h %T "%r" %s %b %b %R %u' --no-ip-validation \
```
I mean like this

Reply:
it will wait 10s between each check of file

Reply:
its worth a try. I'll update my code and test it. I've been running my code for a week. I'll try yours now.

Reply:
does it work for you?

Reply:
Sorry away from my system at the moment. I did make the change and will check my logs for errors when I get back on Saturday.

Reply:
Thanks

Reply:
Managed to access my server tonight. Logs are rotating with no errors however there is another issue with goaccess that I've found. In the rare case where the server loses power or is shut down before the logrotate is due and doesn't restart till past the time of the logrotate is due due the logs will be wiped and not ingested into the goaccess DBs.
So one issue fixed and a new one found :)

Reply:
Please create a new issue for this, thanks!